### ISSUE TITLE
TS type definition for FixedSizeList onScroll is wrong.

### ISSUE THREAD

**Issue by tarb** [micha-lmxt/svelte-window#1]
The type definition for the onScroll callback is 
``` onScroll?: (props: ListOnScrollProps) => any; ```
where ListOnScrollProps is 
``` 
export interface ListOnScrollProps {
    scrollDirection: ScrollDirection;
    scrollOffset: number;
    scrollUpdateWasRequested: boolean;
} 
``` 

The callback seems to pass that deconstructed though, as 
``` onScroll?: (scrollDirection: ScrollDirection, scrollOffset: number, scrollUpdateWasRequested: boolean) => any; ```

I don't know which way you want to go, but the fix should be easy either way. I'm just converting a little react project over that used react-window, so I'll let you know if I find anymore bugs. Thanks for the port 👍

**Comment by tarb** [comment]
Fixed in fe3fbaead9067dd5464c3b21534f13f530282321.

**Comment by micha-lmxt** [comment]
Thank you for the bug report, hope your port runs well. 

I went for the approach to  passing an object as input, since it is the way react-window handles it
